mne/utils.py

```diff
@@ -1172,8 +1172,8 @@ def run_subprocess(command, verbose=None, *args, **kwargs):
     *args, **kwargs : arguments
         Additional arguments to pass to subprocess.Popen.
 
-    Returns
     -------
+    Returns
```

Comment:
FWYW on a somewhat related note I find that checking my own diffs top-to-bottom on GitHub I usually find one or two things I didn't mean to put in there or forgot to change. I usually try to do it before requesting a review, as it cuts down on the review iteration cycles/time a bit.